fbw-a380x/src/wasm/systems/a380_systems_wasm/src/autobrakes.rs

```diff
@@ -4,97 +4,85 @@ use systems_wasm::aspects::{EventToVariableMapping, EventToVariableOptions, Msfs
 use systems_wasm::Variable;
 
 pub(super) fn autobrakes(builder: &mut MsfsAspectBuilder) -> Result<(), Box<dyn Error>> {
-    let options = |options: EventToVariableOptions| {
+    let option_button_press = |options: EventToVariableOptions| {
         options
             .leading_debounce(Duration::from_millis(1500))
             .afterwards_reset_to(0.)
     };
+    let option_knob_event =
+        |options: EventToVariableOptions| options.leading_debounce(Duration::from_millis(1500));
 
+    // Mapping msfs 320 bindings to some 380 modes
+    // LO -> LO  / MED -> BRK 3 / HI -> RTO
     builder.event_to_variable(
         "AUTOBRAKE_LO_SET",
-        EventToVariableMapping::Value(1.),
-        Variable::named("OVHD_AUTOBRK_LOW_ON_IS_PRESSED"),
-        options,
+        EventToVariableMapping::Value(2.),
+        Variable::named("AUTOBRAKES_SELECTED_MODE"),
+        option_knob_event,
     )?;
     builder.event_to_variable(
         "AUTOBRAKE_MED_SET",
-        EventToVariableMapping::Value(1.),
-        Variable::named("OVHD_AUTOBRK_MED_ON_IS_PRESSED"),
-        options,
+        EventToVariableMapping::Value(4.),
+        Variable::named("AUTOBRAKES_SELECTED_MODE"),
+        option_knob_event,
     )?;
     builder.event_to_variable(
         "AUTOBRAKE_HI_SET",
         EventToVariableMapping::Value(1.),
-        Variable::named("OVHD_AUTOBRK_MAX_ON_IS_PRESSED"),
-        options,
+        Variable::named("OVHD_AUTOBRK_RTO_ARM_IS_PRESSED"),
```

Comment:
Why is this different from LO and MED?

Reply:
> why would the AUTO_THROTTLE_DISCONNECT itself re-emmit itself

It's a different event, for use by our other systems. The FBW interface masks the sim A/T disconnect event so nobody else can listen for it.